src/scripts/main.js

```diff
@@ -1,3 +1,251 @@
 'use strict';
 
 // write code here
+document.addEventListener('DOMContentLoaded', function() {
+  const table = document.querySelector('table');
+  const headers = table.querySelectorAll('th');
+  const tableBody = table.querySelector('tbody');
+  const rows = tableBody.querySelectorAll('tr');
+
+  const directions = Array.from(headers).map(function(header) {
```

Comment:
no need to save smth like an array of directions. Anyway, you sort rows based on one chosen column. So save just string 'asc' || 'desc' etc.